File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {

            int current = 0;
            int ans = 0;

            for(int num: nums){
                if(num == 1){
                    current++;
                    ans = std::max(ans  , current);
                }else{
                    current = 0;
                }

            }
         return ans;
    }
};

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:

        max_sum = nums[0]
        curr_sum = 0

        for num in nums:
            curr_sum+=num
            max_sum = max(curr_sum , max_sum)
            if curr_sum <= 0:
                curr_sum = 0
            
        return max_sum
        